bitacross-worker/service/src/prometheus_metrics.rs

```diff
@@ -191,12 +173,6 @@ pub struct EnclaveMetricsReceiver;
 impl ReceiveEnclaveMetrics for EnclaveMetricsReceiver {
 	fn receive_enclave_metric(&self, metric: EnclaveMetric) -> ServiceResult<()> {
 		match metric {
-			EnclaveMetric::SetSidechainBlockHeight(h) => {
-				ENCLAVE_SIDECHAIN_BLOCK_HEIGHT.set(h as i64);
-			},
-			EnclaveMetric::TopPoolSizeSet(pool_size) => {
-				ENCLAVE_SIDECHAIN_TOP_POOL_SIZE.set(pool_size as i64);
-			},
 			EnclaveMetric::TopPoolSizeIncrement => {
 				ENCLAVE_SIDECHAIN_TOP_POOL_SIZE.inc();
```

Comment:
Can this metric be removed together with related code ?

Reply:
it used at two functions `process_top` and `remove_top`, so it seems they used at code we utilise.

```rs
// Update metric
  if let Err(e) = self.ocall_api.update_metric(EnclaveMetric::TopPoolSizeIncrement) {
	warn!("Failed to update metric for top pool size: {:?}", e);
  }
```

do you think this metrics irrelevant anymore?

Reply:
top pool is a sidechain functionality. If we aim for getting rid of sidechain related code from bc-worker it should be removed too.

Reply:
great then I'll review author functionality and will try to remove as much as possible. Thanks

Reply:
Hey @kziemianek, I've managed to remove some functionality with one of [latest commits](https://github.com/litentry/litentry-parachain/pull/2660/commits/fa90de77162c185077079b283417350d5fb7aca6) but it seems whole Author's logic used across the bitacross project and not sure is it relevant or not. :thinking: 

Probably I'd finish with only this chunk of work and merge it. What do you think?